test/app/src/config.ts

```diff
@@ -77,13 +77,15 @@ export function getDefaultConfig(): Config {
 }
 
 export function extendConfig(baseConfig: Config, extraConfig: Config): Config {
+  const useBundledWidget = !!process.env.DIST_ESM;
   const widgetOptions = {
     ...baseConfig.widgetOptions,
     ...extraConfig.widgetOptions,
   };
   const config = {
     ...baseConfig,
     ...extraConfig,
+    useBundledWidget,
```

Comment:
Without this override there can be situation when `useBundledWidget` is loaded as `false` from config storage, but test app is started with `DIST_ESM=1`.
`useBundledWidget` should only depend on current `DIST_ESM` env var.